python/torch_mlir_e2e_test/test_suite/stats.py

```diff
@@ -695,7 +752,6 @@ def VarCorrectionKeepDimModule_basic(module, tu: TestUtils):
 
 
 class VarCorrectionNoneModule(torch.nn.Module):
-
```

Comment:
There are a lot of stylistic changes here unrelated to this PR. Can you make these in a separate PR?